BaseTools/Scripts/PatchCheck.py

```diff
@@ -491,6 +528,19 @@ def error(self, *err):
             print(prefix, line)
             count += 1
 
+    license_format_preflix = 'SPDX-License-Identifier'
+
+    bsd2_patent = 'BSD-2-Clause-Patent'
+
+    bsd3_patent = 'BSD-3-Clause-Patent'
+
+    license_optional_list = ['BSD-2-Clause', 'BSD-3-Clause', 'MIT', 'Python-2.0', 'Zlib']
+
+    Readdedfileformat = re.compile(r'\+\+\+ b\/(.*)\n')

```

Comment:
Similarly, variables are snake case. They are all upper case if they're constants. The rest of the code doesn't seem to be adhering to the last bit, so I would make this lower case snake case.